tests/neg-macros/wrong-owner/Test_2.scala

```diff
@@ -0,0 +1,5 @@
+import scala.annotation.experimental
+
+@experimental
+@wrongOwner
+class Foo // error
```

Comment:
Add missing last line line return to all files.
```suggestion
class Foo // error

```

Reply:
Done